content/docs/clouds/aws/get-started/begin.md

```diff
@@ -110,6 +110,10 @@ $ export AWS_PROFILE="<YOUR_PROFILE_NAME>"
 
 For additional information on setting and using AWS credentials, see [AWS Setup](/registry/packages/aws/installation-configuration/).
 
+{{% notes type="info" %}}
+Manually exporting AWS access keys can be tedious and risky, with frequent plaintext copy-pasting. With Pulumi ESC, you can dynamically retrieve short-lived AWS credentials via OpenID Connect, removing the need for manual exports and enhancing security. Learn more about [Pulumi ESC](/docs/esc/).
```

Comment:
Change "tedious and risky" to "prone to error and a security risk".

Reply:
Thx, added